src/pybind/mgr/orchestrator.py

```diff
@@ -438,7 +438,6 @@ def upgrade_available(self):
         """
         raise NotImplementedError()
 
-
```

Comment:
@jmolmo  just for your info, This change now generates a merge conflict when back-porting to Nautilus.